packages/ketcher-react/src/script/ui/component/form/Select/Select.module.less

```diff
@@ -28,7 +28,7 @@
     color: @color-text-primary;
     font-family: @Inter;
     font-size: @font-size-regular;
-    padding: 0 8px;
+    padding: 4px 8px;
```

Comment:
seems, it's a shared component; changes didn't affect other components?

Reply:
removed this padding change, not needed anymore